source/_docs/configuration/state_object.markdown

```diff
@@ -1,33 +1,59 @@
+| `state.last_updated`  | Time the state or state attributes changed in the state machine in UTC time. This is not updated if neither state nor state attributes changed. Example: `2013-09-17 07:32:51.715874+00:00`.    |
+| `state.attributes`    | A dictionary with extra attributes related to the current state.                                                                                                                                |
+| `state.context`       | A dictionary with extra attributes related to the context of the state.                                                                                                                         |
+
+### About the state
+
+The screenshot shows three lights in different states (the `state.state`): `on`, `off`, and `unavailable`. Each light comes with its own entity state attributes such as `supported_color_modes`, `supported_features`. These attributes have their own state: the state of `supported_color_modes` is `color_temp` and `hs`, the state of the `supported_features` attribute is `4`.
+
+<p class='img'>
+  <img src='/images/integrations/light/state_light.png' alt='Screenshot showing three lights with different states: `on`, `off`, or `unavailable`'>
+  Three lights with different states: `on`, `off`, or `unavailable`.
+</p>
+
+The `state.state` is the heart of the [state object](#about-the-state-object). State holds the information of interest of an entity. For example, if a
+light is on or off, the current temperature, or the amount of energy used. The state object stores 3
+timestamps related to the state: `last_updated`, `last_changed`, and `last_reported`. Each
+entity has exactly one state, and the state only holds one value at a time.
+
+### About entity state attributes
+
+The state only holds one value at a time. However, entities can store related entity state attributes in the state object. For example,
+the state of a light is _on_, and the related attributes could be its
+current brightness and color values. [State change events](/docs/configuration/events/#events-and-state-changes) can be used as triggers.
+The current state can be used in [conditions](/docs/automation/condition/).
+
+### About entity attributes
+
+Entities have some attributes that are not related to its state. There are a few attributes that are available on all entities, such as `friendly_name`. In addition to those, each integration has its own attributes to represent extra state data about the entity. For example, the light integration has attributes for the current brightness and color of the light. When an attribute is not available, Home Assistant will not write it to the state. Entity attributes are optional.
 
 When using templates, attributes will be available by their name. For example `state.attributes.assumed_state`.
 
```

Comment:
I'm unclear about what type of attributes are listed here. Can anyone come up with an introduction to this table? I thought it's the list of common attributes, but I'm not sure now

Reply:
It's common state attributes that may be present for different entity domains. There are some more: `attribution`, `device_class` and `supported_features`.

Reply:
Thank you for explaining! I've added them to the list.